### PR TITLE
Update react-native-libraries.json

### PR DESCRIPTION
**Pull request by gabriel-logan**
# 📝 Why & how

Add two libraries that generate barcodes and qr codes using react native and expo

# ✅ Checklist

- [X ] Added library to **`react-native-libraries.json`**
- [ ] Updated library in **`react-native-libraries.json`**

- [ ] Documented in this PR how to use the feature or replicate the bug.
- [ ] Documented in this PR how you fixed or created the feature.

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -11822,5 +11822,49 @@
...
+    "nameOverride": "React Native Barcode Generator",
+    "examples": [
+      "https://github.com/gabriel-logan/mobile-native-barcode-generator/tree/main/example",
+      "https://github.com/gabriel-logan/mobile-native-barcode-generator/tree/main/docs",
+      "https://github.com/gabriel-logan/mobile-native-barcode-generator/blob/main/README.md"
+    ],
+    "images": ["https://private-user-images.githubusercontent.com/96627244/365431315-31661309-1f46-4d52-ab24-5389a0fbb07e.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU4NTY3OTAsIm5iZiI6MTcyNTg1NjQ5MCwicGF0aCI6Ii85NjYyNzI0NC8zNjU0MzEzMTUtMzE2NjEzMDktMWY0Ni00ZDUyLWFiMjQtNTM4OWEwZmJiMDdlLmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwOVQwNDM0NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mZDNjNzY0ZjA5NTBlNjk1M2UyZTdmYTNhNGZhZTUzYzM0ZWFkMWVkZGUyYWRiYTk3OWQ2ZWQ1Y2U1YzJlNjBkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.jI4AwvdPpvk59nbfLclSsSODFnO9W3m7NWnKg8u12gY"],
```
Comment (Simek) [inline react-native-libraries.json:11835]:
```suggestion
    "images": ["https://private-user-images.githubusercontent.com/96627244/365431315-31661309-1f46-4d52-ab24-5389a0fbb07e.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU4NTY3OTAsIm5iZiI6MTcyNTg1NjQ5MCwicGF0aCI6Ii85NjYyNzI0NC8zNjU0MzEzMTUtMzE2NjEzMDktMWY0Ni00ZDUyLWFiMjQtNTM4OWEwZmJiMDdlLmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwOVQwNDM0NTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mZDNjNzY0ZjA5NTBlNjk1M2UyZTdmYTNhNGZhZTUzYzM0ZWFkMWVkZGUyYWRiYTk3OWQ2ZWQ1Y2U1YzJlNjBkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.jI4AwvdPpvk59nbfLclSsSODFnO9W3m7NWnKg8u12gY"],
```

The private image will stop working when token expires.

File: react-native-libraries.json [hunk 2]
```diff
@@ -11822,5 +11822,49 @@
...
+    "web": false,
+    "expoGo": false,
+    "windows": false,
+    "macos": false,
+    "tvos": false,
+    "visionos": false,
+    "unmaintained": false,
+    "dev": false,
+    "template": false,
+    "newArchitecture": false
```
Comment (Simek) [inline react-native-libraries.json:11847]:
```suggestion
    "android": true
```

False flags can be omitted.

File: react-native-libraries.json [hunk 3]
```diff
@@ -11822,5 +11822,49 @@
...
```
Comment (Simek) [inline react-native-libraries.json:11868]:
```suggestion
    "examples": [
      "https://github.com/gabriel-logan/expo-barcode/tree/main/example"
    ],
    "android": true
```

Entry cleanup.

**Comment by gabriel-logan** [comment]
> Hello @gabriel-logan, thanks for adding new entries to the directory! 👍
> 
> Let's address few nit before we can ship it.

Okay, thanks. It's nice to contribute.

**Comment by Simek** [comment]
Thanks for the quick updates! 👍 

Can you also remove the `nameOverride` fields? We are going to remove them soon, since. like npm we fully operate on npm package names now, refs:
* https://github.com/react-native-community/directory/pull/1264

**Comment by gabriel-logan** [comment]
> Thanks for the quick updates! 👍
> 
> Can you also remove the `nameOverride` fields? We are going to remove them soon, since. like npm we fully operate on npm package names now, refs:
> 
> * [remove "nameOverride" field #1264](https://github.com/react-native-community/directory/pull/1264)

Sure

**Review by Simek — APPROVED** [review]
LGTM! 🚀 

> P.S. See other PR for README links removal exmplanation.

**Pull request merged**